internal/turn/attributes.go

```
@@ -0,0 +1,167 @@
+		Value: value,
+	}
+}
+
+// BuildMalformedTLV creates a raw STUN attribute with a length field that
+// exceeds the actual data. Used for memory disclosure testing.
+// The returned bytes are a raw TLV: [2-byte type][2-byte claimed_length][data...][padding]
+func BuildMalformedTLV(attrType uint16, data []byte, claimedLength uint16) []byte {
+	// TLV header: 4 bytes (type + length)
+	paddedClaimed := claimedLength
+	if paddedClaimed%4 != 0 {
+		paddedClaimed += 4 - (paddedClaimed % 4)
```

golangci-lint (2.12.2)

[error] ineffectual assignment to paddedClaimed

(ineffassign)